helm/oauth2-proxy/values.yaml

```diff
@@ -182,3 +182,33 @@ htpasswdFile:
   # example:
   # entries:
   #  - testuser:{SHA}EWhzdhgoYJWy0z2gyzhRYlN9DSiv
+
+# Configure the session storage type, between cookie and redis
+sessionStorage:
+  # Can be one of the supported session storage cookie/redis
+  type: cookie
+  redis:
+    password: ""
```

Comment:
Are the passwords here not sensitive information that would be better suited to being handled by a Secret?

Reply:
I moved the redis specific arguments into env variables and took the redis-passwords and redis-sentinel-password from a new redis specific secret.